src/development/ui/advanced/gestures.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Taps, drags, and other gestures in Flutter
+title: Taps, drags, and other gestures
```

Comment:
This title is really long already, it seemed reasonable to drop "in Flutter". WDYT @Sfshaza?